libs/langchain-google-cloud-sql-pg/src/vectorStore.ts

```diff
@@ -178,15 +180,86 @@ class PostgresVectorStore extends VectorStore {
     )
   }
 
+  async addVectors(vectors: number[][], documents: DocumentInterface[], options?: { metadatas?: Array<Record<string, any>> | null, ids?: string[] }): Promise<string[] | void> {
```

Comment:
* Use Document instead of the interface (no change needed)
* Only support list as ids as options. No need to support metadata not in the document object.
```suggestion
  async addVectors(vectors: number[][], documents: Document[], options?: { ids?: string[] }): Promise<string[] | void> {
```

Reply:
Done